subprojects/docs/src/docs/userguide/build-cache/build_cache_debugging.adoc

```diff
@@ -196,31 +196,12 @@ Build cache key for task ':compileJava' is 8ebf682168823f662b9be34d27afdf77
 The log shows e.g. which source files constitute the `stableSources` for the `compileJava` task.
 To find the actual differences between two builds you need to resort to matching up and comparing those hashes yourself.
 
-Luckily, you do not have to capture this data yourself - the build scan plugin already takes care of this for you.
-Gradle Enterprise has the necessary data to diagnose the cache miss when comparing two build scans, and will show you the properties that caused a different cache key to be generated.
-
-[.screenshot]
-image::build-cache/adjacent-builds-input-comparison.png[]
-
-There are three ways to compare builds in Gradle Enterprise.
-
-1. Use the scan list to find the scans to compare, selecting them by clicking the respective image:build-cache/compare-small.svg[compare,role="inline-icon",width="30"] or image:build-cache/compare-small-b.svg[compare b,role="inline-icon",width="30"] icon.
-2. From a scan that you wish to compare with another, click the image:build-cache/compare-small.svg[compare,role="inline-icon",width="30"] icon at the top of the scan, then select the second scan by clicking the  image:build-cache/compare-small-b.svg[compare b,role="inline-icon",width="30"] icon from the list of scans.
-3. Click the image:build-cache/previous-next-scan.svg[workspace,role="inline-icon", width="22"] icon in the header of a build scan to show previous and subsequent build scans of the same workspace, then click the image:build-cache/compare-small-b.svg[compare b,role="inline-icon",width="30"] icon next to the build you want to compare.
-
-[.screenshot]
-image::build-cache/adjacent-build-comparison.png[]
-
-It is also possible that task output caching for a cacheable task was disabled.
-When this happens the reason why caching was disabled for the task is reported on the info log level and in the build scan:
-
-[.screenshot]
-image::build-cache/caching-disabled.png[]
+TIP: link:https://docs.gradle.com/enterprise/tutorials/task-inputs-comparison/[Gradle Enterprise] already takes care of this for you; it lets you quickly diagnose a cache miss with the Build Scan(TM) Comparison tool.
 
 [[diagnosing_cache_miss]]
 == Diagnosing the reasons for a cache miss
 
-Having the data from the last section at hand, you should be able to diagnose why the outputs of a certain task were not found in the build cache.
+You should be able to diagnose why the outputs of a certain task were not found in the build cache.
```

Comment:
I think you can leave this sentence as is. It's assuming you are doing the manual process of comparing task input fingerprints.